Tools/cases_generator/generate_cases.py

```diff
@@ -482,15 +477,15 @@ def write_metadata(self, metadata_filename: str, pymetadata_filename: str) -> No
                         case parsing.Macro():
                             self.write_metadata_for_macro(self.macro_instrs[thing.name])
                         case parsing.Pseudo():
-                            self.write_metadata_for_pseudo(self.pseudo_instrs[thing.name])
-                        case _:
-                            typing.assert_never(thing)
+                            self.write_metadata_for_pseudo(
```

Comment:
Sorry, but I really want the `assert_never()` calls to stay (several times below also).

Reply:
Oh I missed some cases